### PR TITLE
Repaired start.sh

### PR DESCRIPTION
**Pull request by vojtakudlacek**
Repaired start.sh to work with mvn clean install

### CODE REVIEW

File: start.sh [hunk 1]
```diff
@@ -1,21 +1,19 @@
...
 
+TOOLS=$path/lib/tools.jar
+RSYNTAXTEXTAREA=$(echo $HOME/.m2/repository/com/fifesoft/rsyntaxtextarea/*/rsyntaxtextarea-*.jar)
+GSON=$(echo $HOME/.m2/repository/com/google/code/gson/gson/*/gson-*.jar)
+BYTEMAN=$(echo $HOME/.m2/repository/org/jboss/byteman/byteman-install/*/byteman-install-*.jar)
+JRD=$(echo $HOME/.m2/repository/java-runtime-decompiler/runtime-decompiler/*/runtime-decompiler-*.jar)
 # launch application
-java -cp ${JAVA_HOME}/lib/tools.jar:\
-$HOME/.m2/repository/com/fifesoft/rsyntaxtextarea/2.6.1/rsyntaxtextarea-2.6.1.jar:\
-$HOME/.m2/repository/io/github/soc/directories/10/directories-10.jar:\
```
Comment (judovana) [inline start.sh]: BYteman gone (likely correct) and directories added? Why so?

File: start.sh [hunk 2]
```diff
@@ -1,21 +1,19 @@
...
-path=`which java`
-# go through symlinks until we are at /usr/lib/jvm/...
-while [[ ${path} != /usr/lib/jvm/* ]]
-do
-path=`readlink $path`
-done
-while [[ $(dirname "$path") != /usr/lib/jvm ]]
-do
+path=`which javac`
+path=`readlink -f $path`
```
Comment (judovana) [inline start.sh]: Much better then orignal one. TY!

File: start.sh [hunk 3]
```diff
@@ -1,21 +1,19 @@
...
 # Locate JDK from $PATH
-path=`which java`
-# go through symlinks until we are at /usr/lib/jvm/...
-while [[ ${path} != /usr/lib/jvm/* ]]
-do
-path=`readlink $path`
-done
-while [[ $(dirname "$path") != /usr/lib/jvm ]]
-do
+path=`which javac`
```
Comment (judovana) [inline start.sh]: I would recomend rename $pah to $javac_home

File: start.sh [hunk 4]
```diff
@@ -1,21 +1,19 @@
...
+path=`readlink -f $path`
 path="$(dirname "$path")"
-done
-export JAVA_HOME=$path
+path="$path/.."
```
Comment (judovana) [inline start.sh]: Maybe better to use dirname second times?

File: start.sh [hunk 5]
```diff
@@ -1,21 +1,19 @@
...
 
+TOOLS=$path/lib/tools.jar
+RSYNTAXTEXTAREA=$(echo $HOME/.m2/repository/com/fifesoft/rsyntaxtextarea/*/rsyntaxtextarea-*.jar)
+GSON=$(echo $HOME/.m2/repository/com/google/code/gson/gson/*/gson-*.jar)
+BYTEMAN=$(echo $HOME/.m2/repository/org/jboss/byteman/byteman-install/*/byteman-install-*.jar)
+JRD=$(echo $HOME/.m2/repository/java-runtime-decompiler/runtime-decompiler/*/runtime-decompiler-*.jar)
```
Comment (judovana) [inline start.sh]: Still the .m2 approach. Good for now, not suitable for distributabel image.

File: start.sh (hunk 1 above)
Comment (vojtakudlacek) [reply start.sh]: Directories is no in pom anymore and byteman is there and it wasen't in the start.sh

Comment (judovana) [reply start.sh]: Yah, sorry. I ment vice versa.  Thank you. for clarification. I keep wondering, what is still used from byteman....

**Comment by judovana** [comment]
Hi Radek!

MErging now. If you have any moire issues with it, please let me know. Will fix it.

**Pull request merged**